bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql

```diff
@@ -17,13 +17,22 @@ WITH sample AS (
       UNNEST(e.events) AS event
     {% else %}
     SELECT
+      {% if app_id == "telemetry" %}
+      *,
+      COUNT(*) OVER (PARTITION BY submission_date, client_id) AS client_event_count
```

Comment:
I feel like it would be good to add this same filtering/counting for glean apps, where we might encounter the same problem. Can you add this?

Reply:
I'd opt for not adding this to glean query at this point. Distribution of event counts per client in this table does not have such long tail like in telemetry. I don't really know what cutoff level would make sense in the long term.

Reply:
I feel like it makes sense to have some kind of sanity check here, just to guard against something weird happening unexpectedly in the future. We can deliberately set the threshold high for now (3,000,000 seems like a fine number to me). That said, I don't feel strongly enough about this to block merging this as-is.